tests/boards/intel_adsp/hda_log/testcase.yaml

```diff
@@ -0,0 +1,7 @@
+tests:
+  boards.intel_adsp:
+    platform_allow: intel_adsp_cavs15 intel_adsp_cavs18 intel_adsp_cavs20 intel_adsp_cavs25
+  boards.intel_adsp.1cpu:
+    platform_allow: intel_adsp_cavs15 intel_adsp_cavs18 intel_adsp_cavs20 intel_adsp_cavs25
+    extra_configs:
+      - CONFIG_MP_NUM_CPUS=1
```

Comment:
Why 1 cpu?  That sounds suspicious, since obviously this needs to work in SMP environments.

Reply:
The real goal was to test with and without KERNEL_COHERENCY just like the hda tests do

Reply:
Which of course I forgot to do the flush on the buffer when its not enabled